sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/spanner/changestreams/it/SpannerChangeStreamIT.java

```diff
@@ -93,6 +103,23 @@ public void before() {
 
   @Test
   public void testReadSpannerChangeStream() {
+    testReadSpannerChangeStreamImpl(pipeline, null);
+  }
+
+  @Test
+  public void testReadSpannerChangeStreamWithAuthorizedRole() {
```

Comment:
Have we tested that if someone breaks the pattern of not using MetadataConfig constructor, then this test fails?

Reply:
Yes. If someone (i.e. Apache Beam code) doesn't use the MetadataConfig constructor to generate the config to access the metadata DB in such a way that breaks access to the metadata DB, this test will catch it.  It is precisely why I modified this test to use a separate metadata database.